### PR TITLE
Update contribute.md

### PR DESCRIPTION
**Pull request by diru1100**
Signed-off-by: diru1100 rohitdandamudi.1100@gmail.com
Fixed issue: #42

### CODE REVIEW

File: common-requests/contribute.md [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
 
-# Contribute
-
-## Process
-
 Contributions will only be considered when linked to an open issue, aligned with the project goals, and attested to by the Contributor (ex. signing off on a commit, accepting the [Developer Certificate of Origin](https://developercertificate.org/)).
 
-To get assigned to an issue, start with on open issue labelled `Good First Issue` in one of the repositories and submit a comment on the issue requesting that it be assigned to you.  To get a quick response, try "at" mentioning a Maintainer or Working Group lead in your comment.
+#### Get assigned
+To get assigned to an issue, start with on open issue labelled `Good First Issue` in one of the repositories and submit a comment on the issue requesting that it be assigned to you.
```
Comment (joewxboy) [inline common-requests/contribute.md:10]: @diru1100 Not your fault, but there's a typo here.  "start with on open issue" should be "start with an open issue".

**Review by joewxboy — CHANGES_REQUESTED** [review]
Could you please fix the typo I noted?  I know it's not your fault, but since we're in there editing the file, we might as well fix it.  Thanks.

**Comment by diru1100** [comment]
> Could you please fix the typo I noted? I know it's not your fault, but since we're in there editing the file, we might as well fix it. Thanks.

Yup, thanks for your review.

**Review by joewxboy — APPROVED** [review]
great!

**Pull request merged**